mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,274 @@
+    <!-- Menu Item Label for Search in Library -->
+    <string name="library_search">Search</string>
+    <!-- Library Page Title -->
+    <string name="library_title">Library</string>
+    <!-- Settings Page Title -->
+    <string name="settings_title">Settings</string>
+    <!-- Content description (not visible, for screen readers etc.): "Menu icon for items on a history item" -->
+    <string name="content_description_history_menu">History item menu</string>
+    <!-- Content description (not visible, for screen readers etc.): "Close button for library settings" -->
+    <string name="content_description_close_button">Close</string>
+    <!-- About content. Sentences are separated by two new lines -->
+    <string name="about_content">
+        %1$s puts you in control.
+        \n\n%1$s is produced by Mozilla. Our mission is to foster a healthy, open Internet.
+    </string>
+
+    <!-- Sessions -->
+    <!-- Title for the list of tabs in the current session -->
+    <string name="tabs_header_title">Current session</string>
+    <!-- Title for the list of tabs in the current private session -->
+    <string name="tabs_header_private_title">Private session</string>
+    <!-- Content description (not visible, for screen readers etc.): Add tab button. Adds a news tab when pressed -->
+    <string name="add_tab">Add Tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Close tab button. Closes the current session when pressed -->
+    <string name="close_tab">Close tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Opens the tab menu when pressed -->
+    <string name="tab_menu">Tab menu</string>
+    <!-- Content description (not visible, for screen readers etc.): Close tab button. Favicon for the session's site -->
+    <string name="favicon_content_description">Bookmark</string>
+    <!-- Button in the current session menu. Deletes the session when pressed -->
+    <string name="current_session_delete">Delete</string>
+    <!-- Button in the current session menu. Saves the session when pressed -->
+    <string name="current_session_save">Save</string>
+    <!-- Button in the current session menu. Opens the share menu when pressed -->
+    <string name="current_session_share">Share</string>
+    <!-- Content description (not visible, for screen readers etc.): Title icon for current session menu -->
+    <string name="current_session_image">Current session image</string>
+    <!-- Text for the button to save a session -->
+    <string name="session_save">Save session</string>
+    <!-- Text for the button to delete a session -->
+    <string name="session_delete">Delete session</string>
+    <!-- Text for the button to delete a single session -->
+    <string name="session_item_delete">Delete</string>
+    <!-- Text to tell the user how many more tabs this session has.
+        The first parameter is how many extra tabs the session has. -->
```

Comment:
Please explain what the placeholder will be replaced with

Reply:
a plus sign and a number

Reply:
Ok, the person who fixes these strings should update the localization comment with that, then. Thanks!